client/public/index.html

```diff
@@ -99,14 +99,10 @@
     <script>
       /**
        * If we modify this script, we must update the CSP hash as follows:
-       * 1. Run `yarn build:client`
-       * 2. Open `client/build/index.html` and locate this (minified) <script> tag.
-       * 3. Copy the inner content of the <script> tag (excluding the script tags).
-       * 4. Open https://csplite.com/csp/sha/ and paste the content.
-       * 5. Copy the `sha256-...` hash.
-       * 6. Open `libs/constants/index.js` and find the current hash in scriptSrcValues.
-       * 7. Remove the old "previous" hash and replace it with the old "current" hash.
-       * 8. Replace the old "current" hash with the new hash (from step 5).
+       * 1. Run `npx jest testing/tests/csp.test.ts --watch`
```

Comment:
Need to ask them to build the changes before running the test. Someone may, not knowing the test is happening on the built file, change the code and will see the test succeed and…

Reply:
I feel like the only reliable way to prevent this kind of issue in general is to add an end-to-end  GitHub Actions test that would actually load the page and then check that the script actually ran. And if it failed, print a message pointing to this instruction. I can make a PR if you would be interested in reviewing it.

Reply:
If we make this test a part of the prod build (make it run at the end), will also solve the issue.

We can also put it at the end of dev build (`yarn dev`). But it'll be a bit annoying when un-minified build is done.

Reply:
> Need to ask them to build the changes before running the test.

Good point, step 1 should be to run `yarn build:client`. At the same time, we should replace `npx` with `yarn` in what is currently step 1.